janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/GraphDatabaseConfigurationInstanceIdTest.java

```diff
@@ -108,11 +117,11 @@ public void instanceIdShouldEqualHostnamePlusSuffix() throws UnknownHostExceptio
     }
 
     @Test
-    public void shouldCreateCustomBackendThreadPoolSize() throws UnknownHostException {
+    public void shouldCreateCustomBackendFixedThreadPoolSize() {
```

Comment:
Nitpick and not necessary to resolve: It seems these tests are placed in the wrong place. Looks like this test class was intended for instance id tests.